gateway/integration_tests/src/main.rs

```diff
@@ -544,6 +547,93 @@ async fn config_test(gw_type: LightningNodeType) -> anyhow::Result<()> {
     .await
 }
 
+/// Test that verifies the various liquidity tools (onchain, lightning, ecash)
+/// work correctly.
+async fn liquidity_test() -> anyhow::Result<()> {
+    devimint::run_devfed_test(|dev_fed, _process_mgr| async move {
+        let federation = dev_fed.fed().await?;
+        let gatewayd_version = util::Gatewayd::version_or_default().await;
+        if gatewayd_version < *VERSION_0_5_0_ALPHA {
+            info!(%gatewayd_version, "Version did not support gateway liquidity management, skipping");
+            return Ok(());
+        }
+
+        let gw_lnd = dev_fed.gw_lnd_registered().await?;
+        let gw_cln = dev_fed.gw_cln_registered().await?;
+
+        let gateways = if let Some(gw_ldk) = dev_fed.gw_ldk_connected().await? {
+            [gw_lnd, gw_ldk].to_vec()
```

Comment:
Can you document why we're omitting `gw_cln` here?

Reply:
Nvm it sounds like we've committed to removing CLN support. Please disregard.

Reply:
It will be removed post v0.5: https://github.com/fedimint/fedimint/issues/6218